spec/support/platform_helpers.rb

```diff
@@ -199,8 +199,7 @@ def selinux_enabled?
 end
 
 def suse?
-  ::File.exist?("/etc/SuSE-release") ||
```

Comment:
This was only necessary on the older SLES boxes. Our new builders should all have the os-release file since we're not building for SLES 11 anymore